jest.config.js

```diff
@@ -4,8 +4,6 @@ const { compilerOptions } = require('./tsconfig')
 module.exports = {
   // A list of reporter names that Jest uses when writing coverage reports
   coverageReporters: ['json', 'html'],
```

Comment:
Remove `moduleDirectories` since it prevents recursive imports within `node_modules`. Specifically:

```
/node_modules/ethereum-cryptography/src/secp256k1.ts:1
    ({"Object.<anonymous>":function(module,exports,require,__dirname,__filename,jest){import { privateKeyVerify } from "secp256k1";
                                                                                      ^^^^^^

    SyntaxError: Cannot use import statement outside a module
```